### PR TITLE
Reorganize Code

### PR DESCRIPTION
**Pull request by williambaibai**
- Reorganized code structure into `src`, `tests` and `data`
- Added script for Travis testing (Could an admin configure Travis integration for this repo? Thanks)

### CODE REVIEW

File: src/scraper.py [hunk 1]
```diff
@@ -7,9 +7,9 @@
 from lxml import html
 import requests
 
-import constants
-from schema import ClassDetailType, ClassType, DayTimeRangeType, GymType
-from utils import generate_id
+import src.constants as constants
```
Comment (kq-li) [inline src/scraper.py]: from the same package you don't need the `src`, think just `.` prefix will work -- @young-k what would you advise here?

**Comment by young-k** [comment]
@meganle can u configure travis

File: tests/tests.py [hunk 2]
```diff
@@ -1,12 +1,14 @@
 import unittest
+import sys 
+sys.path.append('..')
```
Comment (kq-li) [inline tests/tests.py]: what's this for?

**Comment by meganle** [comment]
I believe Travis was already activated! Here's a link to the build for the PR

https://travis-ci.com/cuappdev/uplift-backend/builds/91621051

File: src/scraper.py (hunk 1 above)
Comment (young-k) [reply src/scraper.py]: i think `src.constants` is more standard

**Comment by young-k** [comment]
@williambaibai can you address the comments in this PR?

**Comment by williambaibai** [comment]
Push a new commit which moved tests.py to root directory, reason:
- Ran into an issue where when I run `tests.py` from `tests/`, the directory is inconsistent with `app.py`, and therefore the path to `class_metadata.csv' is different in `constants.py`
- Couldn't figure out a better solution other than moving `tests.py` to the same dir as `app.py`, any ideas on how to work around this?

@young-k @kq-li @DanielNosrati @henrytso

**Comment by young-k** [comment]
@williambaibai – A better solution here definitely exists, can you do more research? You can either find proof that your problem isn't solvable (i.e. a Stack Overflow post), but this isn't a problem that having more people look at will help with.

File: src/scraper.py (hunk 1 above)
Comment (kq-li) [reply src/scraper.py]: guess we might as well use a full `from src.constants import ...` then?

**Comment by kq-li** [comment]
@williambaibai could you give more detail on the issue? where does the path  to `class_metadata.csv` come into play for `tests.py`?

**Comment by kq-li** [comment]
try calling the test file from the root directory, i.e. python tests/tests.py

**Comment by williambaibai** [comment]
> try calling the test file from the root directory, i.e. python tests/tests.py

Tried this and this seems to behave the same as just `cd tests` and then `python tests.py`

**Pull request closed without merging**